corehq/apps/hqmedia/models.py

```diff
@@ -476,9 +476,10 @@ def __init__(self, path, module_id=None, module_name=None, form_id=None,
             raise ValueError("media_class should be a type of CommCareMultimedia")
 
         self.module_id = module_id
```

Comment:
i think it would also be nice to rename this to module_index instead of module_id.
In my head module_id should be module_unique_id only and module_index would be the index of the module in the app.

Reply:
Agreed. I took a look at this and realized `module_id` no longer needs to be here at all. Removed it in f2b566c75de